src/server/migrations/1_create_initial_tables.sql

```diff
@@ -1,19 +1,36 @@
 --sql commands here
 
+CREATE TABLE project (
+  id SERIAL PRIMARY KEY,
+  owner_id TEXT NOT NULL,  
+  name TEXT NOT NULL,
+  description TEXT
+);
+
 CREATE EXTENSION IF NOT EXISTS citext;
 
 CREATE TABLE node (
   id SERIAL PRIMARY KEY,
   label TEXT NOT NULL,
   status TEXT NOT NULL,
   priority TEXT NOT NULL,
-  x INTEGER NOT NULL,
-  y INTEGER NOT NULL
+  x DOUBLE PRECISION NOT NULL,
+  y DOUBLE PRECISION NOT NULL,
+  project_id SERIAL NOT NULL,
```

Comment:
`SERIAL` shouldn't be used in foreign key fields, as it's roughly equivalent to `project_id INTEGER AUTOINCREMENT`.

Foreign key fields are pretty easy to write oneline in Postgres; `project id INTEGER NOT NULL REFERENCES project (id)` -- there's no need to write use the `CONSTRAINT` syntax.